tests/state/state.go

```diff
@@ -183,6 +191,12 @@ type State struct {
 	// The type of client currently being tested.
 	ClientType ClientType
 
+	// The type of Document ACP
+	DocumentACPType DocumentACPType
```

Comment:
thought: I was actually wanting to use this today and was wondering what the status was - I didnt realize it had been sat waiting for review :P

Thanks for moving it!

Reply:
😄 Cheers